src/main/java/seedu/address/logic/commands/ClearItemCommand.java

```diff
@@ -17,6 +18,8 @@
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.setItemList(new ItemList());
+        // clears locations as well
```

Comment:
should recipes be reset as well? or is it automatically reset since all items are removed